src/main/resources/db/migration/V024__Error_notifications.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE error_notifications (
+  id BIGSERIAL PRIMARY KEY,
+  event_id BIGINT NOT NULL REFERENCES process_events(id),
+  notificationId VARCHAR(50) NULL,
+  documentControlNumber VARCHAR(100) NULL,
```

Comment:
documentControlNumber and referenceId would be blank for most of the cases as of now. It will be only be applicable to rescan which has come from ccd.

Reply:
Can notifications concern individual documents?

Reply:
Like @nitinprabhu said, back from ccd when validating number with existing ones. Perhaps something more.

Looks like this table is as minimal as possible. Events table don't concern document control numbers so as per specs I have included it here